lua/autorun/client/cl_sounds.lua

```diff
@@ -0,0 +1,22 @@
+local soundMultiplier = 0.7
+
+local function silenceEmplacement( ent, m )
+    if not ent.sounds then return end
+    for k, sound in pairs(ent.sounds) do
+        local soundLevel = sound:GetSoundLevel() * m
+        sound:SetSoundLevel( soundLevel * m)
+    end
+
+end
+
+local isBasedOn = scripted_ents.IsBasedOn
+
+hook.Remove("OnEntityCreated", "CFC_GredwitchEnhancementsSounds")
+hook.Add( "OnEntityCreated", "CFC_GredwitchEnhancementsSounds", function( ent )
+    local class = ent:GetClass()
+    if class == "gred_emp_base" or isBasedOn( class, "gred_emp_base" ) then
+        timer.Simple(0.1, function()
```

Comment:
Do you know why this works? We're waiting for the entity to be valid, right? What happens when the server is lagging and this takes longer than .1?

Reply:
i don't this will be an issue but if it is we can increase the delay